README.md

```diff
@@ -316,7 +316,36 @@ you can define an error handler:
 
 ```ruby
 # config/initializers/maintenance_tasks.rb
-MaintenanceTasks.error_handler = ->(error) { Bugsnag.notify(error) }
+MaintenanceTasks.error_handler = ->(error, task_context) do
+  task_info = { name: task_content[:task_name] }
+  Bugsnag.notify(error) do |notification|
+    notification.add_tab(:task, task_info)
+  end
+end
+```
+
+The second argument passed to the error handler contains extra information about
+the Task that ran. `task_context[:errored_element]` is the element that was
+being processed, if any, when the Task raised an exception. If you would like
+to pass this object to your exception monitoring service, make sure you
+**sanitize the object to avoid leaking sensitive data** and
+**convert it to a format that is compatible with your bug tracker**.
+
+For example:
+```ruby
+# config/initializers/maintenance_tasks.rb
+MaintenanceTasks.error_handler = ->(error, task_context) do
+  # task_context[:errored_element] is
+  # <CSV::Row "name":"John Doe" "address":"123 Main Street">
+  errored_element = task_context.delete(:errored_element).to_h
+  errored_element['address'] = 'REDACTED'
```

Comment:
Given the example above (the safe default that people can copy paste without spending more time on it), I'm not sure we need to want to have another one.

---

If we want another example, I would go ahead and point out the more Railsy way to do this with a parameter filter, and since we've established [Bugsnag only keeps the id](https://github.com/bugsnag/bugsnag-ruby/blob/f9c539670c448f7f129a3f8be7d412e2e824a357/lib/bugsnag/cleaner.rb#L126-L127).

```ruby
parameter_filter = ActiveSupport::ParameterFilter.new(Rails.application.config.filter_parameters)
MaintenanceTasks.error_handler = ->(error, task_context) do
  errored_element = task_context.delete(:errored_element)
  errored_element = if errored_element.respond_to?(:to_h)
    parameter_filter.filter(errored_element.to_h)
  elsif errored_element.is_a?(ActiveRecord::Base)
    errored_element # bugsnag only sends the id
  end
  task_context[:errored_element] = errored_element
  # ...
end
```